web/src/components/Navigation.vue

```diff
@@ -88,12 +98,14 @@
                 <v-list-item
                   link
                   exact
+                  :style="onShiftPressed"
```

Comment:
wouldn't it make more sense to have two menu items to search for the selected tag vs. add/remove tag to current search? (ideally also showing if it adds or deletes)

Reply:
Having 2 buttons next to each other  instead of the shift click ux?
Personally I think the shift click solution looks cleaner, multiple buttons would look more cluttered.